test/regression/hard_dae.jl

```diff
@@ -221,8 +221,9 @@ refsol = solve(prob, Rodas4(), saveat = 0.1, callback = cb, tstops = [1.0], relt
 for solver in (Rodas4, Rodas4P, Rodas5, Rodas5P, FBDF, QNDF, Rosenbrock23)
     @show solver
     prob = ODEProblem(f, deepcopy(res.zero), (0, 20.0), deepcopy(p_inv))
+    maxiters = solver isa Rosenbrock23 ? 1e8 : 1e5
```

Comment:
is it reasonable that `Rosenbrock23` now takes 1000x more steps than the other solvers?

Reply:
Something is up with that test, I think I need to bring it local and check. It wouldn't be that unreasonable though since it's not B-stable.